go.mod

```diff
@@ -14,35 +14,55 @@ go 1.14
 
 require (
 	github.com/AlekSi/pointer v1.1.0
+	github.com/aws/aws-sdk-go v1.35.15 // indirect
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
+	github.com/go-ini/ini v1.62.0 // indirect
+	github.com/go-logr/logr v0.2.1 // indirect
+	github.com/golang/snappy v0.0.2 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/uuid v1.1.2
+	github.com/googleapis/gnostic v0.4.0 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.2
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/hashicorp/go-version v1.2.1 // indirect
-	github.com/percona-platform/dbaas-api v0.0.0-20201009095023-f103f733767a
+	github.com/imdario/mergo v0.3.11 // indirect
+	github.com/jetstack/cert-manager v1.0.0-beta.0
+	github.com/klauspost/compress v1.11.1 // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
+	github.com/percona-platform/dbaas-api v0.0.0-20201026075036-655507121a69
 	github.com/percona-platform/saas v0.0.0-20201008124851-3c2c6c2ec0ce
-	github.com/percona/percona-backup-mongodb v1.2.0 // indirect
-	github.com/percona/percona-server-mongodb-operator v1.4.0
-	github.com/percona/percona-xtradb-cluster-operator v1.4.0
+	github.com/percona/percona-backup-mongodb v1.3.2 // indirect
+	github.com/percona/percona-server-mongodb-operator v1.5.0
```

Comment:
> Expected result: dbaas-controller does not depend on github.com/percona/percona-xtradb-cluster-operator and github.com/percona/percona-server-mongodb-operator Go modules.

Reply:
percona-server-mongodb-operator is removed in the other PR

Reply:
So I guess we should merge the other PR first? Because this ticket is not done otherwise

Reply:
The "other PR" in question: https://github.com/percona-platform/dbaas-controller/pull/67

Reply:
Actually @percona-csalguero can remove one line from imports and this PR should be ready. No need to wait for the second PR.

Reply:
Did that in 67ba54db1d3a665e646a80204e2cf7e6b4ecfcbd, but that's not enough: we still depend on k8s packages, and on different versions than operators. We should copy a trimmed version of API machinery too.

Reply:
https://jira.percona.com/browse/PMM-6830